src/forms/FormField.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Form, Label} from 'semantic-ui-react';
+/**
+ * The FormField will render a semantic form field that will show a
+ * description below label and error message if given.
+ */
+const FormField = ({
+  id,
+  name,
+  description,
+  required,
+  focused,
+  value,
+  errors,
+  touched,
+  type,
+  placeholder,
+  handleChange,
+  handleBlur,
+  handleFocus,
+  children,
+}) => (
+  <Form.Field required={required}>
+    <label className="noMargin label-inline">{name}:</label>
+    {required && <small className="text-red">(required)</small>}
+    <p className="noMargin">
+      <small>{description}</small>
+    </p>
+    {children ? (
+      children
+    ) : (
+      <Form.Input
+        fluid
+        type={type}
+        name={id}
+        placeholder={placeholder}
+        open={focused}
+        onChange={handleChange}
+        onBlur={e => {
+          handleFocus('');
+          handleBlur(e);
+        }}
+        onFocus={e => handleFocus(id)}
+        value={value}
+        error={
+          touched !== undefined && errors !== undefined && errors.length > 0
+        }
+      />
+    )}
+    {touched && errors && errors.length > 0 && (
+      <Label pointing basic color="red">
```

Comment:
Is there a way to reduce the margin on top of the label so it touches the field instead of floating below it ? 
<img width="219" alt="Screen Shot 2019-09-23 at 1 22 11 PM" src="https://user-images.githubusercontent.com/1334131/65447756-40826180-de05-11e9-8dec-6e10cd1fd4be.png">